modules/load-balancer/variables.tf

```diff
@@ -33,51 +35,56 @@ variable "subnet_ids" {
 
 variable "allow_inbound_from_security_groups" {
   description = "A list of Security Group IDs from which the load balancer will allow incoming HTTP/HTTPS requests. Any time you change this value, make sure to update var.allow_inbound_from_security_groups too!"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
 variable "allow_inbound_from_security_groups_num" {
   description = "The number of Security Group IDs in var.allow_inbound_from_security_groups. We should be able to compute this automatically, but due to a Terraform limitation, if there are any dynamic resources in var.allow_inbound_from_cidr_blocks, then we won't be able to: https://github.com/hashicorp/terraform/pull/11482"
+  type        = number
   default     = 0
 }
 
 variable "default_target_group_arn" {
   description = "The ARN of a Target Group where all requests that don't match any Load Balancer Listener Rules will be sent. If you set this to empty string, we will send the requests to a \"black hole\" target group that always returns a 503, so we strongly recommend configuring this to be a target group that can instead return a reasonable 404 page."
+  type        = string
   default     = ""
```

Comment:
Shouldnt this be null?

Reply:
@Etiene its because its used as an argument to find length and throws an error when its set as null
![Screen Shot 2019-07-08 at 10 35 38 AM](https://user-images.githubusercontent.com/21035422/60800058-61270d80-a16c-11e9-871e-48b6f4d936d6.png)

Reply:
maybe change that to `count = var.default_targed_group_arn == null ? 1 : 0`

Reply:
FWIW, my internal guideline I have been using for when we don't use `null` is if it the value is interpolated in a string like:

```
"${var.name_prefix}-foo"
```

In this case, we can't use `null` because Terraform doesn't support it. Otherwise, we should use `null` as much as possible and make the code changes to support it.